wled00/FX.cpp

```diff
@@ -7229,7 +7229,7 @@ uint16_t mode_2DGEQ(void) { // By Will Tatam. Code reduction by Ewoud Wijma.
   if ((fadeoutDelay <= 1 ) || ((SEGENV.call % fadeoutDelay) == 0)) SEGMENT.fadeToBlackBy(SEGMENT.speed);
 
   for (int x=0; x < cols; x++) {
-    uint8_t  band       = map(x, 0, cols-1, 0, NUM_BANDS - 1);
+    uint8_t  band       = map(x, 0, cols, 0, NUM_BANDS);
```

Comment:
This is in line with the common workaround for know inaccuracies of `map` would be the exact opposite: add `1` to in_max and out_max.

However - as already stated - it would be good to test with common segment sizes before merging.
@adrianschroeter you can simply change the segment size to test your proposed modification.